en/docs/reference/samples/api-development-sample/collaborative-api-development-sample.md

```diff
@@ -58,14 +50,14 @@ The following are the user credentials that you need to use when signing in to t
 
 Follow the instructions below to implement the latter mentioned business scenario using WSO2 API Manger.
 
-1.  Create three [different tenants](https://docs.wso2.com/display/AM260/Managing+Tenants) ( `finance.abc.com, core.abc.com,` and `operations.abc.com` ) for the three departments, with [users](https://docs.wso2.com/display/AM260/Key+Concepts#KeyConcepts-Usersandroles) (John, Tom, and Bob respectively) who can [create APIs](https://docs.wso2.com/display/AM260/Create+and+Publish+an+API) .
+1.  Create three [different tenants]({{base_path}}/administer/multitenancy/managing-tenants) ( `finance.abc.com, core.abc.com,` and `operations.abc.com` ) for the three departments, with [users]({{base_path}}/administer/managing-users-and-roles/introduction-to-user-management) (John, Tom, and Bob respectively) who can [create APIs]({{base_path}}/getting-started/quick-start-guide/#step-1-create-and-publish-an-api) .
```

Comment:
Remove the space between the bracket and ` please. Check the rest of the doc for this issue.
( `finance.abc.com,

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/1762/commits/cb2834840dadcffdece6e356763992c34025e896